l5kit/l5kit/dataset/ego.py

```diff
@@ -54,8 +54,10 @@ def __init__(
             render_context=render_context,
             history_num_frames=cfg["model_params"]["history_num_frames"],
             history_step_size=cfg["model_params"]["history_step_size"],
+            history_step_time=cfg["model_params"]["history_delta_time"] * cfg["model_params"]["history_step_size"],
```

Comment:
I'm fine in leaving it as it is for now, but we should really decide if we want all this flexibility. I think we should just remove step_size all together honestly..